tests/llmobs/test_experiments.py

```diff
@@ -90,6 +90,50 @@ def test_dataset_create_delete(llmobs):
     llmobs._delete_dataset(dataset_id=dataset._id)
 
 
+def test_dataset_as_dataframe(llmobs):
+    dataset = llmobs.create_dataset(name="test-dataset-3", description="A third test dataset")
+    dataset._records = [
+        DatasetRecord(
+            input_data=[{"role": "system", "content": "i am machine"}, {"role": "user", "content": "hello"}],
+            expected_output="label",
+        )
+    ]
+    df = dataset.as_dataframe()
```

Comment:
as a sanity check can we assert some things on df so that we're sure this works as intended?

Reply:
done!